src/seedu/addressbook/parser/Parser.java

```diff
@@ -11,17 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.addressbook.commands.AddCommand;
-import seedu.addressbook.commands.ClearCommand;
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```

Comment:
!!!

Reply:
"While this is more readable, this is bad practice as it would affect performance and potentially pollute the namespace. `java.util` is a big library!"
Quoted from @juxd

Reply:
Hi guys,

If you look at `seedu.addressbook.commands`, `Parser.java` actually uses almost ALL the commands inside. So it does not pollute the namespace here. Plus, IntelliJ optimised this for me... 

@monmanuela `java.util` is not relevant here

Cheers!